language/move-lang/src/lib.rs

```diff
@@ -171,13 +176,19 @@ pub fn move_compile(
         Ok(res) => res,
     };
 
+    let pprog = if !compilation_env.flags().testing_set() {
+        unit_test::filter_test_members::program(&mut compilation_env, pprog)
+    } else {
+        pprog
+    };
+
```

Comment:
I would probably stick this in move_run at in the expansion pass. If someone manually invokes `move_continue_up_to`, they won't get the filtering